### PR TITLE
add symptoms and transmission to details page

### PR DESCRIPTION
**Pull request by attwad**

### CODE REVIEW

**Comment by attwad** [comment]
[Page to display a full case details](https://trello.com/c/OcIt0g8b/216-page-to-display-a-full-case-details)

File: verification/curator-service/ui/src/components/ViewCase.tsx [hunk 1]
```diff
@@ -297,6 +297,53 @@ function CaseDetails(props: CaseDetailsProps): JSX.Element {
...
+                    <RowHeader title="Symptoms" />
+                    <RowContent
+                        content={props.c.symptoms?.values?.join(', ')}
+                    />
+
+                    <RowHeader title="Pre existing conditions" />
+                    <RowContent
+                        content={
+                            props.c.preexistingConditions?.values?.join(', ') ||
+                            ''
```
Comment (allysonjp715) [inline verification/curator-service/ui/src/components/ViewCase.tsx:320]: nit: don't think `|| ''` is necessary

Comment (attwad) [reply verification/curator-service/ui/src/components/ViewCase.tsx:320]: indeed

Comment (attwad) [reply verification/curator-service/ui/src/components/ViewCase.tsx:320]: oh well no, typescript complains... meh

**Pull request merged**